source/Locales/DistributiveLattice/Isomorphism.lagda

```diff
@@ -0,0 +1,83 @@
+--------------------------------------------------------------------------------
```

Comment:
Notice that a lattice isomorphism amounts to simply an equivalence that preserves and reflects order.

Reply:
Noted down in #269.